ui/app/styles/core/buttons.scss

```diff
@@ -52,97 +52,97 @@ $button-box-shadow-standard: 0 3px 1px 0 rgba($black, 0.12);
     box-shadow: none;
   }
 
-  @each $name, $pair in $colors {
```

Comment:
What in here is bulma that needs to be commented out? This just looks like sass 🤔